packages/core/src/app/customer/CheckoutButton.tsx

```diff
@@ -34,3 +52,12 @@ export default class CheckoutButton extends PureComponent<CheckoutButtonProps> {
         return <div id={containerId} />;
     }
 }
+
+export default withCheckout(({
+    checkoutState,
+}: CheckoutContextProps) => {
+    const { data: { getCheckout } } = checkoutState;
+    const { outstandingBalance } = getCheckout() || {};
+
+    return { outstandingBalance };
```

Comment:
Could we put them inside a function for readability?

Reply:
Sorry, didn't understand what you mean